protocol/backend/backend.go

```diff
@@ -137,7 +137,7 @@ func NewBackend(cfg *Config) (Backend, error) {
 	}
 
 	if cfg.Environment == common.Development {
-		defaultTimeoutDuration = time.Minute
+		defaultTimeoutDuration = 75 * time.Second
```

Comment:
Increased this because, on slow cloud VMs, the XMR locking phase is taking 40 seconds (combined with longer eth transaction times after  abandoning instamine).